### PR TITLE
reuploading documentation

### PR DESCRIPTION
**Pull request by takdeerSM**
### Pull Request Overview

This pull request adds the documentation again. 

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Takdeer S M <s_m.takdeer@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/s_m.takdeer/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+
+## VitalTrack: Wearable Fitness and Health Monitor
+A wearable device that tracks steps, heart rate, and location, with data visualization and smartphone integration.
+
+
+:::info
+
+Author: TAKDEER S M
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:8]: Please write your full name.

File: docs/project/2024/s_m.takdeer/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Sensors: Includes an accelerometer for step tracking, a heart rate sensor for monitoring pulse, and a GPS module for location tracking.
+Display/Indicators: A small display or LED indicators provide visual feedback and data display directly on the device.
+Communication Module: Handles data transmission between the fitness tracker and a smartphone.
+These components are interconnected, with the microcontroller at the core processing inputs from the sensors and sending information to the display and through the communication module.
+
+## Log
+
+Week 6 - 12 May: 
+Week 7 - 19 May: 
+Week 20 - 26 May:
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:33]:
```suggestion
### Week 6 - 12 May

### Week 7 - 19 May

### Week 20 - 26 May
```

File: docs/project/2024/s_m.takdeer/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## Log
+
+Week 6 - 12 May: 
+Week 7 - 19 May: 
+Week 20 - 26 May: 
+
+
+## Hardware
+
+    Microcontroller: Raspberry Pi Pico W
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:38]:
```suggestion
### Microcontroller: Raspberry Pi Pico W
```

Comment (alexandruradovici) [reply docs/project/2024/s_m.takdeer/index.md:38]: Please use the same for all subtitles.

File: docs/project/2024/s_m.takdeer/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    Small Display or LED Indicators
+Functionality: Displays real-time data such as step count, heart rate, and GPS maps. LED indicators can provide simple feedback like heart rate zones or connectivity status.
+Reason for Selection: A display enriches the user interface by providing a visual readout of data, while LED indicators offer minimalistic, power-saving feedback.
+
+## Schematics
+Place your KiCAD schematics here (not provided in text format).
+
+## Bill of Materials
+
+Device	                |  Usage	             |  Price
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:63]: Please format the table correctly and verify that it displays using `npm start`.

File: docs/project/2024/s_m.takdeer/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Raspberry Pi Pico W	    | The microcontroller    |35 RON  [https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html]
+Accelerometer	        |Step tracking	         |13 RON  [https://www.optimusdigital.ro/ro/senzori-senzori-inertiali/97-modul-accelerometru-cu-3-axe-adxl345.html?gad_source=1&gclid=Cj0KCQjwudexBhDKARIsAI-GWYWJSj7gQn3KVIODZsQ75yYTZjC3MPmSpByy592yeU-X4ncr1yfCf5gaApJ9EALw_wcB]
+Heart rate sensor	    |Heart rate monitoring   |5 RON  [https://www.optimusdigital.ro/en/others/12594-heart-rate-sensor-module.html]
+GPS module	            |Location tracking       |45 RON  [https://www.optimusdigital.ro/en/gps/105-gy-neo6mv2-gps-module.html?gad_source=1&gclid=Cj0KCQjwudexBhDKARIsAI-GWYVr6IyMZX1Ynbjm0SX9Md8QoaR7rUFRcTRJWrdmEJVxdJ-6XjGoY2gaAk_XEALw_wcB]
+Small display/LEDs      |Data display	         |10-20 RON  [https://www.optimusdigital.ro/en/73-led-displays]
+
+
+## Software
+
+Library	            | Description	                        |Usage
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:73]: Please format the table correctly.

File: docs/project/2024/s_m.takdeer/index.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+Sensor Libraries	|Libraries for interfacing with sensors.|	Handle data acquisition from accelerometer, heart rate sensor, and GPS.
+Display Driver Libraries|	Drivers for managing output to displays.|	Manage visual output on the device's display.
+Communication Libraries 	|Libraries to enable Wi-Fi connectivity.	|Facilitate data transmission to smartphones or the cloud.
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [https://github.com/topics/workout-tracker]
+2. [https://users.rust-lang.org/t/initiative-tracker-first-full-practice-project/97159]
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:85]: Please use the correct markdown syntax for links and give them titles.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please close the previous pull request.

File: docs/project/2024/s_m.takdeer/index.md (hunk 1 above)
Comment (takdeerSM) [reply docs/project/2024/s_m.takdeer/index.md:8]: This is my name in all of my id cards and documents

Comment (alexandruradovici) [reply docs/project/2024/s_m.takdeer/index.md:8]: S M? Ok

File: docs/project/2024/s_m.takdeer/index.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
+
+## VitalTrack: Wearable Fitness and Health Monitor
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:2]:
```suggestion
# VitalTrack: Wearable Fitness and Health Monitor
```

File: docs/project/2024/s_m.takdeer/index.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+| GPS module            | Location tracking         | [45 RON](https://www.optimusdigital.ro/en/gps/105-gy-neo6mv2-gps-module.html?gad_source=1&gclid=Cj0KCQjwudexBhDKARIsAI-GWYVr6IyMZX1Ynbjm0SX9Md8QoaR7rUFRcTRJWrdmEJVxdJ-6XjGoY2gaAk_XEALw_wcB) |
+| Small display/LEDs    | Data display              | [10-20 RON](https://www.optimusdigital.ro/en/73-led-displays) |
+
+
+
+## Software
+
+| Library                   | Description                                | Usage                                             |
+|---------------------------|--------------------------------------------|---------------------------------------------------|
+| Pico SDK                  | Official SDK for Raspberry Pi Pico.        | Fundamental for programming and hardware interaction. |
```
Comment (alexandruradovici) [inline docs/project/2024/s_m.takdeer/index.md:82]: Links?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**